### PR TITLE
Remove NewRelic

### PR DESCRIPTION
**Pull request by rsanheim**
**Story card:** [ch1541](https://app.clubhouse.io/simpledotorg/story/1541/remove-new-relic)

We are switching to Datadog, so replacing custom new relic metrics w/ Datadog Statsd calls.

### CODE REVIEW

File: config/initializers/new_relic.rb [hunk 1]
```diff
@@ -1,58 +0,0 @@
...
-  [Patient.singleton_class, :discarded_record],
-  [Patient.singleton_class, :invalid_record],
-  [Patient.singleton_class, :create_new_record],
-  [Patient.singleton_class, :update_existing_record],
-  [Patient.singleton_class, :return_old_record],
-
-  [MergePatientService, :merge],
-  [BloodPressure.singleton_class, :merge],
-  [Appointment.singleton_class, :merge],
-  [PrescriptionDrug.singleton_class, :merge]
```
Comment (rsanheim) [inline config/initializers/new_relic.rb:47]: @simpledotorg/server-developers Not sure how much any of this was used or looked at -- we can look into custom Datadog spans to replace this for what we still want to monitor.

File: app/controllers/api/v3/exotel_call_sessions_controller.rb [hunk 2]
```diff
@@ -70,13 +68,9 @@ def respond_in_plain_text(status, text = "")
     render plain: text, status: status
   end
 
-  def report_http_status
```
Comment (timcheadle) [inline app/controllers/api/v3/exotel_call_sessions_controller.rb:73]: I take it DataDog gives us these error code rates?

File: spec/controllers/api/v3/exotel_call_sessions_controller_spec.rb [hunk 3]
```diff
@@ -22,11 +22,7 @@
                               digits: patient.phone_numbers.first.number,
                               CallType: "call-attempt",
                               CallSid: SecureRandom.uuid}
-
-        expect(response).to have_http_status(200)
```
Comment (timcheadle) [inline spec/controllers/api/v3/exotel_call_sessions_controller_spec.rb:26]: This looks like it should stay?

**Review by timcheadle — APPROVED** [review]
Approving to unblock, just left a few comments. Assuming all of the custom event metrics are no longer needed.

File: app/controllers/api/v3/exotel_call_sessions_controller.rb (hunk 2 above)
Comment (rsanheim) [reply app/controllers/api/v3/exotel_call_sessions_controller.rb:73]: Yup.

File: spec/controllers/api/v3/exotel_call_sessions_controller_spec.rb (hunk 3 above)
Comment (rsanheim) [reply spec/controllers/api/v3/exotel_call_sessions_controller_spec.rb:26]: Oops yeah. Adding back.

**Pull request merged**